File: cases/taint/csharp/local-overwrite-positive/LocalOverwritePositive.cs
namespace DataFlowBench;

static class LocalOverwritePositive
{
    static int dfb_source() // DFB-SOURCE: local-overwrite-input
    {
        return 1;
    }

    static void dfb_sink(int value) { } // DFB-SINK: local-overwrite-sink

    static void Run()
    {
        int value = dfb_source();
        value = value; // DFB-WITNESS: local-overwrite-preserved
        dfb_sink(value);
    }
}

File: cases/taint/go/local-overwrite-positive/local_overwrite_positive.go
package dataflowbench

func dfb_source() int { // DFB-SOURCE: local-overwrite-input
	return 1
}

func dfb_sink(value int) {} // DFB-SINK: local-overwrite-sink

func run() {
	value := dfb_source()
	value = value // DFB-WITNESS: local-overwrite-preserved
	dfb_sink(value)
}
